inst/pages/30_differential_abundance.qmd

```diff
@@ -112,7 +129,7 @@ threshold of 10%.
 
 ```{r prep-daa-data}
 # Agglomerate by genus and subset by prevalence
-tse <- subsetByPrevalent(tse,
+tse <- subsetByPrevalentFeatures(tse,
```

Comment:
subsetByPrevalent is the new name of the function. Update your mia directly from Github